examples/python/easy_start_demo/client.py

```diff
@@ -0,0 +1,98 @@
+"""
+Author: Zhongying Wang
+Email: kerbalwzy@gmail.com
+DateTime: 2019-08-13T23:30:00Z
+PythonVersion: Python3.6.3
+"""
+import os
+import sys
+import time
+import grpc
+
+# add the `demo_grpc_dps` dir into python package search paths
+BaseDir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, os.path.join(BaseDir, "demo_grpc_pbs"))
+
+from demo_grpc_pbs import demo_pb2, demo_pb2_grpc
+
+ServerAddress = "127.0.0.1:23334"
+ClientId = 1
+
+
+# 简单模式
+# Simple Method
+def simple_method(stub):
+    print("--------------Call SimpleMethod Begin--------------")
+    req = demo_pb2.Request(Cid=ClientId, ReqMsg="called by Python client")
```

Comment:
We tend not to use abbreviations in the codebase. Please prefer `request` to `req`, `response` to `resp`, etc. Here and throughout.